internal/command/ssh/console_windows.go

```diff
@@ -1,30 +1,71 @@
-//go:build windows
-// +build windows
-
 package ssh
 
 import (
-	"fmt"
 	"os"
 
 	"github.com/Azure/go-ansiterm/winterm"
 )
 
-func init() {
```

Comment:
init() is a magic method that runs when the package is imported... do we need to replicate that behavior now that it won't happen implicitly?

Reply:
So admittedly I don't fully understand the lower-level console output code, particularly the Windows magical bits, but that initial `init` was enabling virtual terminal processing under Windows, in the belief that there were no consequences for doing so.

Unfortunately, that universal hack broke prompt input such that arrows spat out their control characters. Further, I just started using PowerShell more heavily, and after running an SSH console I was no longer able to arrow up through command history. When I removed that original hack on a whim, arrows were handled correctly in prompts, and I was able to navigate command history again.

Since the original hack was only needed for `ssh console`, I now wrap that console invocation in calls that enable virtual terminal processing/input, then restore the original values when the console exits.

Not sure if there's a better pattern for this in Go. Please let me know if there is--I can't decide whether my dislike of this code is due to Go or my lack of knowledge. :)

I've tested spinning up an SSH console and launching a new app in the same terminal. In all cases, including arrowing through command history and prompt input, my terminal behaved as expected.

Reply:
oh perfect, if you've tested and things are working that's good enough for me. Thanks!